docs/site/debugger.md

```diff
@@ -179,3 +179,47 @@ Most importantly, make sure you have `cider/cider-nrepl` as a dependency, and `c
 ```sh
 clojure -Sdeps '{:deps {nrepl/nrepl {:mvn/version,"0.8.3"},cider/cider-nrepl {:mvn/version,"0.25.8"}}}' -m nrepl.cmdline --middleware "[cider.nrepl/cider-middleware]"
 ```
+
+## Passing options to the REPL JVM
+
+There are times when clojure debugging tools are not enough or not right for the job.
+This is usually true when use an (open source) Java library and you want to set some brake points in Java code.
+For those cases and others, you need to start the JVM in debug mode.
+And for that you nee
+
+Tipical use cases:
+
+* Change java logger configuration for the REPL via java system properties: e.g `-Dorg.slf4j.simpleLogger.defaultLogLevel=TRACE`
+* Enable JVM debugger, change VM memory size, etc
+
+Calva supports passing environment variables via `jacInEnv`.
+You can set that option inside VSCode `settings.json` file.
+
+You can configre global `settings.json` file or a project wide version.
+I think it's best to put these options inside your project, inside `<project-root>/.vscode/settings.json`.
+This is because we don't know how the changes will impact other VSCode projects that start a JVM.
+See documentation for `settings.json` here: https://code.visualstudio.com/docs/getstarted/settings .
+
+The bellow snippet configures `JAVA_TOOL_OPTIONS` environment variable.
+This variable is read by the JVM and the configuration is applied accordingly.
+
+NOTE: You can of course pass other env variables here.
+
+.vscode/settings.json
+```json
+{
+    "calva.jackInEnv": {
+        "JAVA_TOOL_OPTIONS": "${env:JAVA_TOOL_OPTIONS} -Dorg.slf4j.simpleLogger.defaultLogLevel=TRACE -Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=7896"
+    }
+}
+```
+
+Once you saved the file, the next time you `Jack in` the project, you should see something like:
+
+```shell
+clojure -Sdeps '{:deps {nrepl/nrepl {:mvn/version,"0.8.3"},cider/cider-nrepl {:mvn/version,"0.26.0"}}}' -A:debug -m nrepl.cmdline --middleware "[cider.nrepl/cider-middleware]"
+WARNING: Implicit use of clojure.main with options is deprecated, use -M
```

Comment:
This warning will be removed in the next release, so we can remove it from here.

```suggestion
```

Reply:
I removed the warning.